packages/@guardian/source-react-components-development-kitchen/components/divider/Divider.tsx

```diff
@@ -35,10 +37,40 @@ const decideSpace = (
 	}
 };
 
+const decideSize = (size: Size, displayText?: string) => {
```

Comment:
To be honest, I think the previous method was simpler to follow. By adding this function you're coupling `decideText` in with `size` in a way that wasn't there before.

Can we remove this change from this PR for now and focus in on the `cssOverrides` change - that way we can merge it through quickly and then have some breathing space to take stock around the larger api discussion.